packages/x-charts-pro/src/FunnelChart/useFunnelChartProps.ts

```diff
@@ -199,6 +199,8 @@ export const useFunnelChartProps = (props: FunnelChartProps) => {
 
   const clipPathProps: ChartsClipPathProps = {
     id: clipPathId,
+    // Else the stroke will be cut off when variant is outlined
+    offset: { top: 1, bottom: 1 },
```

Comment:
We can also try to use a polyfill for the `stroke-align` property 😆 

https://stackoverflow.com/a/75403983/24269134

Reply:
What about removing clip-path for funnel?